server/neptune/workflows/internal/lyft/audit_workflow_runner_wrapper.go

```diff
@@ -0,0 +1,62 @@
+package lyft
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/activities"
+	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/config/logger"
+	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/deploy/revision/queue"
+	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/deploy/terraform"
+	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/job"
+	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/root"
+	"go.temporal.io/sdk/workflow"
+)
+
+type auditActivities interface {
+	AuditJob(ctx context.Context, request activities.AuditJobRequest) error
+}
+
+type AuditWorkflowRunnerWrapper struct {
+	Activity auditActivities
+	queue.TerraformWorkflowRunner
+}
+
+func (w *AuditWorkflowRunnerWrapper) Run(ctx workflow.Context, deploymentInfo terraform.DeploymentInfo) error {
```

Comment:
Why are we wrapping it like this? We only did that in the past to not conflate with upstream changes. It harms readability imo.

Reply:
I originally planned on following what we had for `project_command_runner` to be able to upstream in the future. But, after our offline chat, I changed this approach and instead directly emit events in the receiver.